src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/AuthUserDTO.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
+
+class AuthUserDTO(
```

Comment:
_nitpick_
Would we benefit from `id` here?

Reply:
it's currently not used in the UI, we can add it easily later if required.

Reply:
Thinking out loud -- it feels weird to expose username/auth source as they're both used for identification and not the ID. Not a big deal :)